src/components/c-breadcrumb.vue

```diff
@@ -56,10 +56,17 @@
 </script>
 
 <style lang="scss">
+  @use '../setup/scss/variables';
+  @use '../setup/scss/mixins';
+  @use '../setup/scss/extends';
```

Comment:
- [ ] Why you defined the extends by the @use rule when you need the @import afterwords?

Reply:
@claudio147 the import of the list reset extends is temporary and has already been removed in the branch https://github.com/valantic/vue-template/pull/174 which you also have for review.